File: Modul 3/Soal 5/PRAK205-2210817310006-Ahmad Muhsin Zein.c
#include <stdio.h>

int main()
{
    for(int i = 0; i <= 1; i++){
        int a,b,al,k,l;
        scanf ("%d %d", &a,&b);
        al=sqrt(pow(b,2)-pow(a,2));
        k=a+b+al;
        l=0.5*al*a;
        printf("\nAlas\t\t= %d cm\n",al);
        printf("Tinggi\t\t= %d cm\n",a);
        printf("Keliling\t= %d cm\n",k);
        printf("Luas\t\t= %d cm^2\n\n",l);
    }
}

File: Modul 3/Soal 5/PRAK205-2210817310006-Ahmad Muhsin Zein.py
import math
a,b = input().split()
a =float(a)
b =float(b)
al=(b*b)-(a*a)
alas=math.sqrt(al)
k=a+b+alas
l=0.5*alas*a
print("Alas =", round(alas),"cm")
print("Tinggi =", round(a),"cm")
print("Keliling =", round(k),"cm")
print("Luas =", round(l),"cm^2\n")

a1= int(input())
b1= int(input())
al1=(b1*b1)-(a1*a1)
alas1=math.sqrt(al1)
k1=a1+b1+alas1
l1=0.5*alas1*a1
print("Alas =", round(alas1),"cm")
print("Tinggi =", round(a1),"cm")
print("Keliling =", round(k1),"cm")
print("Luas =", round(l1),"cm^2\n")